File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.cpp
class Solution {
public:
    int largestAltitude(vector<int>& gain) {
        int max1 = 0;
        int temp = max1;
        for(int i = 0; i < gain.size(); i++){
            temp  +=  gain[i];
            max1 = max(max1,temp);
        }
        return max1;
    }
};

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.java
class Solution {
    public int largestAltitude(int[] gain) {
        int maxx = 0;
        int temp = 0;
        for(int i =0; i< gain.length; i++){
            temp += gain[i];
            maxx = maxx > temp ? maxx: temp;
        }
        return maxx;
    }
}